src/web/jcc/client/components/CommandControl/CommandControl.tsx

```diff
@@ -2616,16 +2607,9 @@ export default class CommandControl extends React.Component {
             end = taskPacketsTimeline.end as string;
         }
 
-        this.api
-            .getTaskPackets(taskPacketsTimeline.start as string, end)
-            .then((taskPackets) => {
-                this.setTaskPackets(taskPackets);
-                taskData.updateTaskPacketsLayers(taskPackets);
-                success("Getting Task Packets...");
-            })
-            .catch((err) => {
-                console.error("Task Packets Timeline Submission Error:", err);
-            });
+        taskData.update(taskPacketsTimeline.start as string, end).then(() => {
```

Comment:
Since this function was modified and does not have a function comment yet we should add one.

Reply:
This one (handleKeepEndDateCurrentToggle) does seem to already have a function comment.  Did you mean a different method?